src/box/txn_limbo.c

```diff
@@ -649,10 +662,66 @@ txn_limbo_read_demote(struct txn_limbo *limbo, int64_t lsn)
 	return txn_limbo_read_promote(limbo, REPLICA_ID_NIL, lsn);
 }
 
+/**
+ * Check that some synchronous transactions have gathered quorum and
+ * write a confirmation entry of the last confirmed transaction.
+ */
+void
+txn_limbo_confirm(struct txn_limbo *limbo)
```

Comment:
The function is never used outside of `txn_limbo.c` which means we can make it `static`. Lets do this.

Reply:
Fixed.